environment.yml

```
@@ -5,8 +5,8 @@ channels:
 dependencies:
   - python=3.10
```

Oops, wait, need to actually update this in the environment.yml file too :sweat_smile:
```suggestion
  - python=3.11
```